terminus/geometry/arc.py

```diff
@@ -195,7 +242,7 @@ def heading_at_offset(self, offset):
         if offset > self.length():
             raise ValueError("Offset ({0}) is greater than segment length ({1})".format(offset, self.length()))
         angular_offset = math.copysign(offset, self._angular_length) * 180 / (math.pi * self._radius)
-        return self._theta + angular_offset
+        return (self._theta + angular_offset) % 360
```

Comment:
create a method `_clip_angle_to_360(angle)` and call it from all the places that you are doing `% 360`.